src/ui/server/helpers/mappings/map-currencies/as-radio-options/index.test.ts

```diff
@@ -1,50 +1,88 @@
-        value: currency4.isoCode,
-      },
-      ALTERNATIVE: {
-        text: CONTRACT_POLICY[POLICY_CURRENCY_CODE][ALTERNATIVE_POLICY_CURRENCY_CODE].TEXT,
-        value: CONTRACT_POLICY[POLICY_CURRENCY_CODE][ALTERNATIVE_POLICY_CURRENCY_CODE].ID,
-      },
-    };
-
-    expect(result).toEqual(expected);
+  describe(`policy ${ALTERNATIVE_POLICY_CURRENCY_CODE}`, () => {
+    it('should return an object of supported countries options and an ALTERNATIVE option', () => {
+      const result = mapCurrenciesAsRadioOptions(mockCurrencies, ALTERNATIVE_POLICY_CURRENCY_CODE);
+
+      const [currency0, currency1, currency2, currency3, currency4] = mockCurrencies;
+
+      const expected = {
+        [currency0.isoCode]: {
+          text: nameAndIsoCodeText(currency0.name, currency0.isoCode),
+          value: currency0.isoCode,
+        },
+        [currency1.isoCode]: {
+          text: nameAndIsoCodeText(currency1.name, currency1.isoCode),
+          value: currency1.isoCode,
+        },
+        [currency2.isoCode]: {
+          text: nameAndIsoCodeText(currency2.name, currency2.isoCode),
+          value: currency2.isoCode,
+        },
+        [currency3.isoCode]: {
+          text: nameAndIsoCodeText(currency3.name, currency3.isoCode),
+          value: currency3.isoCode,
+        },
+        [currency4.isoCode]: {
+          text: nameAndIsoCodeText(currency4.name, currency4.isoCode),
+          value: currency4.isoCode,
+        },
+        ALTERNATIVE: {
+          text: FIELDS[ALTERNATIVE_POLICY_CURRENCY_CODE].TEXT,
+          value: ALTERNATIVE_POLICY_CURRENCY_CODE,
+        },
+      };
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe(ALTERNATIVE_CURRENCY_CODE, () => {
+    it('should return an object of supported countries options and an ALTERNATIVE option', () => {
+      const result = mapCurrenciesAsRadioOptions(mockCurrencies, ALTERNATIVE_CURRENCY_CODE);
+
+      const [currency0, currency1, currency2, currency3, currency4] = mockCurrencies;
+
+      const expected = {
+        [currency0.isoCode]: {
+          text: nameAndIsoCodeText(currency0.name, currency0.isoCode),
+          value: currency0.isoCode,
+        },
+        [currency1.isoCode]: {
+          text: nameAndIsoCodeText(currency1.name, currency1.isoCode),
+          value: currency1.isoCode,
+        },
+        [currency2.isoCode]: {
+          text: nameAndIsoCodeText(currency2.name, currency2.isoCode),
+          value: currency2.isoCode,
+        },
+        [currency3.isoCode]: {
+          text: nameAndIsoCodeText(currency3.name, currency3.isoCode),
+          value: currency3.isoCode,
+        },
+        [currency4.isoCode]: {
+          text: nameAndIsoCodeText(currency4.name, currency4.isoCode),
+          value: currency4.isoCode,
+        },
+        ALTERNATIVE: {
+          text: FIELDS[ALTERNATIVE_POLICY_CURRENCY_CODE].TEXT,
+          value: ALTERNATIVE_CURRENCY_CODE,
+        },
```

Comment:
This seems to be duplicate twice in this test, could we have at the top e.g `expectedCurrencies` array and then just spread in each test?